paper/paper.md

```diff
@@ -89,6 +90,6 @@ To download easyaccess you can follow the provided options or clone the source c
 
 # Acknowledgments
 
-The DES Data Management System is supported by the National Science Foundation under Grant NSF AST 07-15036 and NSF AST 08-13543
+The DES Data Management System is supported by the National Science Foundation under Grant NSF AST 07-15036 and NSF AST 08-13543. 
 
 # References
```

Comment:
Do we need the standard DES references?

Reply:
I've added them, they don't need to be in the md file, they are added when the pdf is compiled using paper.bib